lib/Core/StatsTracker.cpp

```diff
@@ -252,17 +252,17 @@ StatsTracker::StatsTracker(Executor &_executor, std::string _objectFilename,
       klee_error("%s", sqlite3ErrToStringAndFree("Can't set options for database: ", zErrMsg).c_str());
     }
 
+    // create table
+    writeStatsHeader();
```

Comment:
You should only move the header here. Otherwise writeStatsLine can commit a non open transaction

Reply:
I did :see_no_evil: